django/__init__.py

```diff
@@ -5,14 +7,22 @@
 __version__ = get_version(VERSION)
 
 
-def setup():
+def setup(set_prefix=True):
     """
     Configure the settings (this happens as a side effect of accessing the
     first setting), configure logging and populate the app registry.
+    Optionally define the thread-local urlresolvers script prefix if
```

Comment:
Chop "Optionally?" (it's always done if `set_prefix` is True which is part of the sentence)
"define" -> "set"?